conda_smithy/lint_recipe.py

```diff
@@ -19,10 +19,7 @@
 import github
 import ruamel.yaml
 
-from conda_build.metadata import (
-    ensure_valid_license_family,
-    FIELDS as cbfields,
-)
+from conda_build.metadata import ensure_valid_license_family, FIELDS as cbfields
```

Comment:
Can we stop going back and forth on this? This was changed in https://github.com/conda-forge/conda-smithy/pull/1003 and now it's reverted here